docs/src/appendix/cheatcodes/start_spoof.md

```diff
@@ -72,7 +72,7 @@ fn test_spoof() {
     
     // Cancel tx_info.transaction_hash mocking by setting the field to `None`
     // All other fields of `TxInfo` remain unchanged
-    start_spoof(contract_address, TxInfoMockTrait::default());
+    start_spoof(CheatTarget::One(contract_address), TxInfoMockTrait::default());
     dispatcher.store_tx_hash();
     
     let tx_hash = dispatcher.get_stored_tx_hash();
```

Comment:
Theoretically, the line below
```
assert(tx_hash == tx_hash_before_mock, 'tx_hash should not be mocked anymore');
```
would fail when executed, since the message `tx_hash should not be mocked anymore` is 36 characters long and won't fit in short string

It'd be problematic if someone was to copy the example and try to run it themselves

Reply:
fixed